docs/UserGuide.md

```diff
@@ -67,205 +72,219 @@ and efficient Graphical User Interface interaction.
-* At least one field has to be specified in the edit command.
+* Note that `EMAIL` can only be used for suppliers while `ADDRESS` can only be used for warehouses.
+* At least one of the optional parameters have to be specified in the input.
 * Edited supplier or warehouse must be different from one started with.
-* No two suppliers or two warehouses can share the same name.
+* `TYPE` specified should be either `s` for supplier or `w` for warehouse.
 
 Examples:
 
-* `edit ct/s i/1 n/Alice p/85236417 r/Largest Supplier e/alicekoh@example.com` Edits the first supplier in the displayed supplier list to be named _Alice_ with contact number of _85236417_ and remark of _Largest Supplier_. In addition, email field of supplier is to be edited as _alicekoh@example.com_.
-* `edit ct/w i/2 n/Bob p/67851234 r/Largest Warehouse addr/Jurong Street 11` Edits the second warehouse in the displayed warehouse list to be named _Bob_ with contact number of _67851234_ and remark of _Largest Warehouse_. In addition, address field of warehouse is to be edited as _Jurong Street 11_.
+* `edit ct/s i/1 n/Alice p/85236417 r/Largest supplier e/alicekoh@example.com` : Edits the name, phone, remark and email
+  of the first supplier in the list of suppliers to be `Alice`, `85236417`, `Largest supplier` and
+  `alicekoh@example.com`.
+* `edit ct/w i/2 n/Bob p/67851234 r/Largest warehouse addr/Jurong Street 11` : Edits the name, phone, remark and
+  address of the second warehouse in the list of warehouses to be `Bob`, `67851234`, `Largest warehouse`
+  and `Jurong Street 11`.
 
-### Finding medical product associated with warehouses / suppliers: `find`
+### Finding relevant supplier(s) or warehouse(s): `find`
 
-Finds all suppliers or warehouses whose name, remark and/or products sold/stored matches the provided argument keywords.
+Finds all supplier(s) or warehouse(s) whose name, remark and/or products sold/stored matches the provided keywords.
 
 Format: `find ct/TYPE [n/NAME…​] [pd/PRODUCT_NAME…​] [r/REMARK…​]`
 
-* Keywords specified are case-insensitive.
-* The TYPE specified should be one of these values: s/w.
-* Any combination of the name, product and remark prefix can be provided but at least one of the prefix for name,
-product or remark must be specified.
+* `NAME`, `PRODUCT_NAME` and `REMARK` are case-insensitive.
+* `TYPE` specified should be either `s` for supplier or `w` for warehouse.
+* Any combination of the `NAME`, `PRODUCT_NAME` and `REMARK` parameters can be provided but at least one of the prefix
+  must be specified.
 
 Examples:
 
-* `find ct/s pd/masks` Displays all the suppliers that sell masks.
-* `find ct/w pd/PANADOL SUSP r/biggest` Displays the biggest warehouse that stores products matching either PANADOL or
-SUSP.
+* `find ct/s pd/masks` : Displays all the suppliers that sell `masks`.
+* `find ct/w pd/panadol r/biggest` : Displays the biggest warehouse that stores products matching `panadol`.
```

Comment:
I try to have 2 warehouses having product 'Panadol' and remark 'biggest', it seems to work well. So I guess the warehouse here should be "warehouses" and maybe we can also use 'product with name' in place of 'products matching'?